### ISSUE TITLE
[Screen Reader - Elixir - Review Settings] JAWS doesn't announce the position of column header on table under 'Review Settings' tab.

### ISSUE THREAD

**Issue by mohit406** [microsoft/fluentui#17887]
**User Experience:**
Properly defined information & relationship of an UI element helps people with different disabilities, by allowing user agents to provide proper navigation & content according to the needs of individual users.

Note: User credentials should NOT be included in the bug.

**Repro Steps:**

1. Open URL: https://elixirppe.microsoft.com/home#  and login with valid creds.
2. Navigate to Settings button at the bottom of screen and select it.
3. Navigate to all controls under Review settings tab.
4. Verify that JAWS announces the position of column header on table or not.

**Actual Result:**
JAWS doesn't announce the position of column header on table under 'Review Settings' tab. There are four column header present on table.

**Expected Result:**
JAWS should announce the position of column header on table under 'Review Settings' tab. It should be read as (Column header Review group name 1 of 4, Column header created 2 of 4, Column header Related top area 3 of 4 and Column header Actions 4 of 4).

**Note:**
This issue does not exists with Narrator & NVDA.
Same issue exists throughout the application where this type of control appears.

**MAS Reference:**
[MAS 4.2.1 - Object Information](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7BEEE97BBD-9990-48A0-8B03-6EA775581AA8%7D&file=MAS%204.2.1%20%E2%80%93%20Object%20Information.docx&wdLOR=cBC823FF2-F30B-4029-A49F-90FCC0AB9FEB&action=default&mobileredirect=true&cid=96658fb6-7f9e-4bb7-9f68-1d3e7894faf8)

**Reference Links:**

- [Accessibility Insights!](https://apc01.safelinks.protection.outlook.com/?url=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906834622&sdata=0WbVE4dkxMrMm9bbxE1O5xshTfo1%2B0frnzlbmWVeHTs%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://apc01.safelinks.protection.outlook.com/?url=https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906839611&sdata=D5fhtDtz6f9O9hGT6oPoowXq0L/TCfaejH73thazOvA%3D&reserved=0). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process related queries.

Test Environment :

- Build: Version 20H2 (OS Build 19042.685) 
- Browser: Chrome (Version 88.0.4324.150 (Official Build) (64-bit))
- Screen reader: JAWS(2020)
- #URL:https://elixirppe.microsoft.com/home#

**Comment by mohit406** [comment]
A11yMAS, A11ySev2, Accessibility, Benchmark, Feature-HCL-Elixir-Feb21, HCL, MAS4.2.1, Win10-Chrome

![MAS4 2 1_JAWS doesn't announce the position of column header on table under 'Review Settings' tab](https://user-images.githubusercontent.com/82201935/115537755-c522b380-a2b8-11eb-8e3d-be73f32e8d5e.jpg)

**Comment by mohit406** [comment]
This bug has been copied from: https://msazure.visualstudio.com/One/_workitems/edit/9370652

**Comment by tomi-msft** [comment]
Duplicate Issue: https://github.com/microsoft/fluentui/issues/17886
